static/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -47,3 +47,18 @@ The performance of NTFS degrades while handling large volumes of files in a
 directory.
 [Here](https://stackoverflow.com/questions/197162/ntfs-performance-and-large-volumes-of-files-and-directories)
 is the resource for reference.
+
+## Enabling paging with `less`
+
+By default, DVC tries to use [Less](<https://en.wikipedia.org/wiki/Less_(Unix)>)
+as pager for the output of `dvc pipeline show`. Windows doesn't have the less
+command available however. Fortunately, there is a easy way of installing `less`
+via [Chocolatey](https://chocolatey.org/) (please install the tool first):
+
+```dvc
+$ choco install less
+```
+
+`less` can be installed in other ways, just make sure it's available in
+`cmd`/Powershell, where you run dvc. (This usually means adding the directory
```

Comment:
```suggestion
`cmd`/Powershell, where you run `dvc`. (This usually means adding the directory
```

Reply:
Actually I can commit this 🙂 One min...

Reply:
Ah no, sorry. I don't have permission to push to your fork. Please ![image](https://user-images.githubusercontent.com/1477535/70668248-ea64d380-1c27-11ea-9959-02d0a0a01856.png)

Reply:
(p.s. FYI you can allow upstream repo maintainers to push to the branch in the PR settings.)

Reply:
@jorgeorpinel let's merge and fix it in-place? :) it's too minor to do a cycle.

Reply:
True, I can definitely do that!

Reply:
Fixed in 99558335.